cli/src/pcluster/validators/directory_service_validators.py

```diff
@@ -88,7 +89,7 @@ def _validate(self, password_secret_arn: str, region: str):
             if service == "ssm":
                 resource = arn_components[5].split("/")[0]
 
-            if service == "secretsmanager" and resource == "secret" and region != "us-isob-east-1":
+            if service == "secretsmanager" and resource == "secret":
                 AWSApi.instance().secretsmanager.describe_secret(password_secret_arn)
             elif service == "ssm" and resource == "parameter" and region == "us-isob-east-1":
```

Comment:
Why is the `and region == "us-isob-east-1"` required?

Reply:
backward compatibility

Reply:
If you are referring to line 94, then the reason to keep it is backward compatibility since in 3.5.1 we provided the support for password stored in SSM Parameter